tests/test_models.py

```diff
@@ -2235,6 +2239,7 @@ def setUp(self):
         self.user = UserFactory()
         self.auth = Auth(user=self.user)
         self.project = ProjectFactory(creator=self.user, description='foobar')
+        ensure_licenses()
```

Comment:
I believe `set_backends` is `False` in that call-- so the licenses never get generated. 

I'll move this to a try/except in the factory.